pkgs/tools/admin/drawterm/default.nix

```diff
@@ -2,36 +2,50 @@
 , lib
 , fetchgit
 , xorg
+, pkg-config
+, wayland-scanner
+, pipewire
+, wayland
+, wayland-protocols
+, libxkbcommon
+, wlr-protocols
+, useConfig ? "unix"
 }:
 
 stdenv.mkDerivation rec {
   pname = "drawterm";
-  version = "unstable-2023-03-05";
+  version = "unstable-2023-06-27";
 
   src = fetchgit {
     url = "git://git.9front.org/plan9front/drawterm";
-    rev = "ed9cff5a4c39322744c4708699c9ae6651b7c9ab";
-    sha256 = "LM6UnggoxKC3e6xOlHYk9VFF99Abbdmp37nuUML8RgI=";
+    rev = "36debf46ac184a22c6936345d22e4cfad995948c";
+    sha256 = "ebqw1jqeRC0FWeUIO/HaEovuwzU6+B48TjZbVJXByvA=";
   };
 
-  buildInputs = [
-    xorg.libX11
-    xorg.libXt
-  ];
+  nativeBuildInputs = {
+    linux = [ pkg-config wayland-scanner ];
+    unix = [];
```

Comment:
Is this depending on meta.platforms? If so we can just use that instead.

Reply:
Not quite. The "unix" and "linux" here refer to the upstream drawterm build targets, not meta.platforms. The names are perhaps slightly misnomers. "unix" = x11 + OSS, "linux" = wayland + pipewire. So for nixos, which still defaults to X11. it makes more sense to use the "unix" upstream target.